dll/win32/shlwapi/shlwapi.spec

```diff
@@ -568,7 +568,8 @@
 568 stdcall AssocQueryStringW(long long wstr wstr ptr ptr)
 569 stdcall ChrCmpIA(long long)
 570 stdcall ChrCmpIW(long long)
-571 stdcall ColorAdjustLuma(long long long)
+571 stdcall -noname -version=0x600+ SHInvokeCommandWithFlagsAndSite(ptr ptr ptr ptr long str)
+@ stdcall ColorAdjustLuma(long long long)
```

Comment:
`@` means auto-generated ordinal, as what does this end up when compiling for `0x600+` ?

Reply:
I'm not sure what you are asking here. ColorAdjustLuma does not have a stable ordinal so we don't care what it ends up as.

Reply:
I am unsure what ordinal this gets now, I am afraid in the GCC case this is going to end up clashing (which should be impossible,..)

Reply:
```
859 stdcall -noname -version=0x600+ foo()
@ stdcall bar()
860 stdcall -noname -version=0x500+ baz()
```

Ends up leaving a gap with nothing assigned to 859 with GCC. bar ends up as 863 (first free non-ordinal spot in this case). I don't see a problem with this.